receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -150,8 +154,105 @@ func (prw *prometheusRemoteWriteReceiver) parseProto(contentType string) (promco
 }
 
 // translateV2 translates a v2 remote-write request into OTLP metrics.
-// For now translateV2 is not implemented and returns an empty metrics.
+// translate is not feature complete.
 // nolint
-func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, _ *writev2.Request) (pmetric.Metrics, promremote.WriteResponseStats, error) {
-	return pmetric.NewMetrics(), promremote.WriteResponseStats{}, nil
+func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *writev2.Request) (pmetric.Metrics, promremote.WriteResponseStats, error) {
+	var (
+		badRequestErrors []error
```

Comment:
can we just use a single error, and use errors.Join to append new errors?  e.g.
```golang
var err error
// then, when we want to append an error:
err = errors.Join(err, fmt.Errorf("missing metric name in labels"))
```

Reply:
no problems with that, I was just doing the same as in the Prometheus codebase :)